nixos/modules/system/boot/stage-1.nix

```
@@ -686,6 +687,24 @@ in
       '';
     };
 
+    boot.initrd.stage1Greeting = mkOption {
+      type = types.str;
+      default = "<<< ${config.system.nixos.distroName} Stage 1 >>>";
+      defaultText = lib.literalExpression ''"<<< ''${config.system.nixos.distroName} Stage 1 >>>"'';
```

```suggestion
      defaultText = literalExpression ''"<<< ''${config.system.nixos.distroName} Stage 1 >>>"'';
```

Might as well, since this file uses `with lib;`.